test/danetest.c

```diff
@@ -22,12 +22,16 @@
 #ifndef OPENSSL_NO_ENGINE
 #include <openssl/engine.h>
 #endif
+#include "testutil.h"
+#include "test_main_custom.h"
 
-#include "../e_os.h"
+#include "e_os.h"
```

Comment:
We have different preferences...  Me, I'd prefer it if we included e_os.h with an explicit path and didn't have to add `..` all over `test/build.info`.  Your preference seems to be the opposite.